experiments/long_runs/land_region.jl

```diff
@@ -51,6 +51,7 @@ time_interpolation_method = LinearInterpolation(PeriodicCalendar())
 regridder_type = :InterpolationsRegridder
 context = ClimaComms.context()
 device = ClimaComms.device()
+ClimaComms.init(context)
```

Comment:
can remove, duplicate

Reply:
I'm still confused about this - I don't see anywhere in the code where we initialize our comms contexts... but I'll remove this for now

Reply:
oh im sorry! I mis read this, I was reviewing it too quickly. I was confusing it with `context = ClimaComms.context()` on line 52.

Reply:
I'll add them in as part of another PR - see #811